oceans/sw_extras/sw_extras.py

```diff
@@ -855,11 +855,43 @@ def zmld_so(s, t, p, threshold=0.05, smooth=None):
     smooth : int
         size of running mean window, to smooth data.
 
+    Return
+    ------
+    zuml : int
+        Depth of the upper mixed layer
+
+    Examples
+    --------
+    >>> from oceans import sw_extras as swe
+    >>> import numpy as np
+    >>> s = np.array([
+        34.0991,  34.0978,  34.0978,  34.1006,  34.1023,  34.1041,
```

Comment:
As you can see from the travis failure, this syntax is not allowed in docstrings.  See [this](https://github.com/pyoceans/python-oceans/blob/master/oceans/sw_extras/sw_extras.py#L250---251) for an example on how to do line continuations on docstrings.

BTW: you can run the tests locally, before send the PR with the command `py.test` issue from the package root directory (where `setup.py` is).